### PR TITLE
Remove and ignore all generated .soy.js and .gss.css files

### PR DESCRIPTION
**Pull request by bdero**
I verified that they're all regenerated. There were actually a couple of versioned generated filed here that were no longer used and no longer have their respective source files versioned.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -66,4 +66,5 @@ html_app/index.html
 .vagrant/
 static/
 
-
+*\.soy\.js
+*\.gss\.css
```
Comment (carsongee) [inline .gitignore]: Can we not just ignore folders called `gen`? or are there things we want to keep in them?

**Comment by bdero** [comment]
Yep, just tried it and committed; same exact result!

**Comment by carsongee** [comment]
cool, :tangerine:

**Comment by bdero** [comment]
:clap:

**Pull request merged**